app/Http/Controllers/AdministrationController.php

```diff
@@ -44,29 +95,84 @@ public function nominationsSearch(Request $request)
                 $data,
                 function ($var) use ($search, $data) {
                     return AdministrationController::array_any(
-                            $data,
-                            function ($alias) use ($search) {
+                        $data,
+                        function ($alias) use ($search) {
                                 return str_contains(strtolower($alias['nominee_first_name']), strtolower($search));
                             }
-                        ) ||
+                    ) ||
                         AdministrationController::array_any(
                             $data,
                             function ($alias) use ($search) {
-                                return str_contains(strtolower($alias['nominee_last_name']), strtolower($search));
-                            }
+                                    return str_contains(strtolower($alias['nominee_last_name']), strtolower($search));
+                                }
                         ) ||
                         AdministrationController::array_any(
                             $data,
                             function ($alias) use ($search) {
-                                return str_contains(strtolower($alias['nominee_email']), strtolower($search));
-                            }
+                                    return str_contains(strtolower($alias['nominee_email']), strtolower($search));
+                                }
                         );
                 }
             );
             return view('administration/nominations', ["nominations" => $data, "search" => $search]);
         }
     }
 
+    private static function getNomineesData()
+    {
+        $nominees = Nominee::all();
+        if (count($nominees) === 0) {
+            self::FillNomineesTable();
+        }
+        $faculties = Faculty::all();
+        $data = [];
+
+        foreach ($nominees as $nominee) {
+            $nominee['faculty'] = $faculties->find($nominee['faculty_id']);
+            $nominee['achievements'] = implode('; ', unserialize($nominee['achievements']));
+            array_push($data, $nominee);
+        }
+        return $data;
+    }
+
+    private static function FillNomineesTable()
+    {
+        $allowedCategories = [1, 2, 3, 4, 5, 6];
+        $nominations = Nomination::whereIn('category_id', $allowedCategories)->get()->toArray();
```

Comment:
Nemusis vsude pouzivt toArray, je to jak linq, pracuj s interfacem kolekce a pak executuj jeji vytvoreni

Reply:
Musím, jinak to neumím

Reply:
Categories::all()?... a sesbirat ids? kamo...

Reply:
C# 
List == IEnumerable a s enumerable bezne pracujes aniz bys udelal toList

Lara
array == Collection a s tou muzes taky normalne pracovat aniz bys udelal toArray

Reply:
![image](https://user-images.githubusercontent.com/49519258/226912412-98bee377-6540-48b2-831d-2ced396a754a.png)

Reply:
https://laravel.com/docs/10.x/collections#available-methods

Reply:
https://laravel.com/docs/10.x/collections#method-merge

Reply:
Tohle uz fakt predelavat nehodlam

Reply:
Funguje to

Reply:
> Tohle uz fakt predelavat nehodlam

Jen point do budoucna, co se ti bude hodit snad i v praci, udelej to priste poradne, zavcas, ne nakokot ze to budou muset ostatni predelavat a na posledni chvili  a nebudes muset nic predelavat :)